### ISSUE TITLE
running example using "within xx seconds"

### ISSUE THREAD

**Issue by yang-cao** [sameeragarwal/blinkdb#10]
When I run simple SQL queries like

Select approx_count(_) from table1 where A = 'xx'    *_within 0.1 seconds**

I run into an error: Parse Error: mismatched input 'within' expecting EOF near ''table1''

Does blinkdb support clauses like "within xx seconds" as shown in the paper as running examples?

**Comment by wangchuan2008888** [comment]
as i digging into source code , only simplewith operator is supported...
This confuse me a lot.

**Comment by sunriseLe** [comment]
@yang-cao ,I have also encountered the same problem with you
I use BlinkDB CLI,and run simple SQL queries like:
select count(1) from src within 1 seconds;
I run into an error:
**FAILED: Parse Error: line 1:25 Failed to recognize predicate 'within'. Failed rule: 'kwInner' in join type specifier
**
Have you solved this problem?Thanks.

**Comment by wangchuan2008888** [comment]
@sunriseLe I donnot think there has that function. as deep in the source code ,only samplewith operator and approx_sum,approx_avg,approx_count implement.

**Comment by yang-cao** [comment]
@sunriseLe Yes, I agree with @wangchuan2008888 I guess such functions may not be supported by BlinkDB (but I didn't look into the code).

**Comment by sunriseLe** [comment]
So，why are the statements written in papers?

**Comment by sunriseLe** [comment]
I can only run statements such as samplewith operator and approx_sum,approx_avg,approx_count.I want to query with time and error bounds,but I failed! @yang-cao @wangchuan2008888

**Comment by qingzma** [comment]
Yeah, I think only samplewith, approx_sum, approx_avg, and approx_count are implemented. I could not find any more modification in the code.

**Comment by pyongjoo** [comment]
FYI, we are actively developing a related open-source AQP system at https://github.com/mozafari/verdictdb

Right now, our engine has the ability to progressively aggregate data (with possibly interactive visual updates), but please file an issue if you have some uses cases for "within XXX seconds".

_Disclaimer: I am the main developer of this project._
